test/cli/data/gapic/batch/common.yaml

```diff
@@ -0,0 +1,19 @@
+common:
+  batch_apis: '*'
+  api_config_pattern: ${GOOGLEAPIS}/gapic/api/artman_${API_SHORT_NAME}.yaml
+  artman_language_yaml: ${GOOGLEAPIS}/gapic/lang/common.yaml
+  git_repos:
```

Comment:
I think I would prefer not have an empty dictionary set here. I _want_ the dictionary to fail hard with a not found if you specify one that does not exist in that permutation.